docs/models.yml

```diff
@@ -1190,382 +1531,506 @@ list_of_speakers:
     on_delete: CASCADE
     equal_fields: meeting_id
+    restriction_mode: B
   tag_ids:
     type: relation-list
     to: tag/tagged_ids
     equal_fields: meeting_id
+    restriction_mode: B
   projection_ids:
     type: relation-list
     to: projection/content_object_id
     equal_fields: meeting_id
+    restriction_mode: B
   meeting_id:
     type: relation
     to: meeting/topic_ids
     required: true
+    restriction_mode: B
 
 motion:
-  id: number
-  number: string
+  id:
+    type: number
+    restriction_mode: A
+  number:
+    type: string
+    restriction_mode: A
   number_value:
     type: number
     description: The number value of this motion. This number is auto-generated and read-only.
     read_only: true
+    restriction_mode: D
   sequential_number:
     type: number
     description: The (positive) serial number of this motion. This number is auto-generated and read-only.
     read_only: true
+    restriction_mode: C
   title:
     type: string
     required: true
-  text: HTMLStrict
+    restriction_mode: A
+  text:
+    type: HTMLStrict
+    restriction_mode: C
   amendment_paragraph_$:
     type: template
     fields: HTMLStrict
-  modified_final_version: HTMLStrict
-  reason: HTMLStrict
+    restriction_mode: C
+  modified_final_version:
+    type: HTMLStrict
+    restriction_mode: C
+  reason:
+    type: HTMLStrict
+    restriction_mode: C
   category_weight:
     type: number
     default: 10000
-  state_extension: string
-  recommendation_extension: string
+    restriction_mode: C
+  state_extension:
+    type: string
+    restriction_mode: C
+  recommendation_extension:
+    type: string
+    restriction_mode: C
   sort_weight:
     type: number
     default: 10000
+    restriction_mode: C
   created:
     type: timestamp
     read_only: true
+    restriction_mode: C
   last_modified:
     type: timestamp
     read_only: true
+    restriction_mode: C
 
   lead_motion_id:
     type: relation
     to: motion/amendment_ids
     equal_fields: meeting_id
+    restriction_mode: C
   amendment_ids:
     type: relation-list
     to: motion/lead_motion_id
     equal_fields: meeting_id
+    restriction_mode: C
   sort_parent_id:
     type: relation
     to: motion/sort_child_ids
     equal_fields: meeting_id
+    restriction_mode: C
   sort_child_ids:
     type: relation-list
     to: motion/sort_parent_id
     equal_fields: meeting_id
+    restriction_mode: C
   origin_id:
     type: relation
     to: motion/derived_motion_ids  # Note: The related motions may not be in the same meeting
+    restriction_mode: B
   derived_motion_ids:
     type: relation-list
     to: motion/origin_id  # Note: The related motions may not be in the same meeting
-  all_origin_ids: number[]  # Calculated: All parents (origin_id), grandparents, etc. of this motion.
-  all_derived_motion_ids: number[]  # Calculated: All children (derived_motion_ids), grandchildren, etc. of this motion.
+    restriction_mode: B
+  all_origin_ids:
```

Comment:
`restriction_mode` is missing here and below

Reply:
The two fields are not documented in the wiki. Which group should I use?

https://github.com/OpenSlides/OpenSlides/wiki/motion-restrictions

Reply:
Oh, the field `forwarding_tree_motion_ids` was removed and replaced with both of them; needs a fix in the wiki, too.

Reply:
So maybe it would be a good idea if the fields where only defined in the models.yml 😉

done